fbw-a32nx/src/systems/instruments/src/EWD/NewPseudoFWC.ts

```diff
@@ -0,0 +1,2200 @@
+            ),
+            whichCodeToReturn: () => [0],
+            codesToReturn: ['000036001'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000060': { // SPEED BRK
+            flightPhaseInhib: [1, 8, 9, 10],
+            simVarIsActive: this.speedBrakeCommand,
+            whichCodeToReturn: () => [![6, 7].includes(this.fwcFlightPhase.get()) ? 1 : 0],
+            codesToReturn: ['000006001', '000006002'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000200': { // PARK BRK
+            flightPhaseInhib: [3, 4, 5, 6, 7, 8],
+            simVarIsActive: this.parkBrake,
+            whichCodeToReturn: () => [0],
+            codesToReturn: ['000020001'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000040': { // NW STRG DISC
+            flightPhaseInhib: [],
+            simVarIsActive: this.nwSteeringDisc,
+            whichCodeToReturn: () => [this.engine1State.get() > 0 || this.engine2State.get() > 0 ? 1 : 0],
+            codesToReturn: ['000004001', '000004002'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000160': { // PTU ON
+            flightPhaseInhib: [],
+            simVarIsActive: this.hydPTU,
+            whichCodeToReturn: () => [0],
+            codesToReturn: ['000016001'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000210': { // RAT OUT
+            flightPhaseInhib: [],
+            simVarIsActive: this.ratDeployed.map((v) => v > 0),
+            whichCodeToReturn: () => [[1, 2].includes(this.fwcFlightPhase.get()) ? 1 : 0],
+            codesToReturn: ['000021001', '000021002'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000070': { // IGNITION
+            flightPhaseInhib: [],
+            simVarIsActive: this.engSelectorPosition.map((v) => v === 2),
+            whichCodeToReturn: () => [0],
+            codesToReturn: ['000007001'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000540': { // PRED W/S OFF
+            flightPhaseInhib: [1, 10],
+            simVarIsActive: this.predWSOn.map((v) => !v),
+            whichCodeToReturn: () => [[3, 4, 5, 7, 8, 9].includes(this.fwcFlightPhase.get()) || SimVar.GetSimVarValue('L:A32NX_TO_CONFIG_NORMAL', 'bool') ? 1 : 0],
+            codesToReturn: ['000054001', '000054002'],
+            memoInhibit: () => false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
```

Comment:
I think inhibit and isactive are mixed up here

This is the condition on the current master

`   '0000540': // PRED W/S OFF
        {
            flightPhaseInhib: [],
            simVarIsActive: !!(predWSOn === 0 && ![1, 10].includes(flightPhase)),
            whichCodeToReturn: [[3, 4, 5, 7, 8, 9].includes(flightPhase) || toconfig === 1 ? 1 : 0],
            codesToReturn: ['000054001', '000054002'],
            memoInhibit: false,
            failure: 0,
            sysPage: -1,
            side: 'RIGHT',
        },`

Reply:
Well ok, seems to have the same effect, but for the sake of consistency :D

Reply:
Fair, I think I saw the "it has the same effect" and refactored it subconsciously to avoid creating another `MappedSubject`, but it's been changed now!